install/upgrade/update_to_dev.php

```diff
@@ -181,7 +181,16 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addfield($table, 'plugin_orion_tasks_id', 'integer', ['after' => 'dl_filename']);
    $migration->addKey($table, 'entities_id', 'entities_id');
    $migration->addPostQuery("UPDATE `$table` SET `parse_status` = 'parsed'");
-   $migration->addPostQuery("UPDATE `$table` SET `filename` = CONCAT('" . addslashes(GLPI_DOC_DIR) . "', `filename`)");
+
```

Comment:
I found a solution of the query commented. Remove the folowing add an put again the line 184 with this sentence... 

```sql
UPDATE `$table` SET `filename` = CONCAT('flyvemdm/package/', IF(`filename` REGEXP '^/', SUBSTR(`filename`, 2), `filename`)) WHERE `filename` NOT REGEXP '^flyvemdm/package/';
```
With this you will got all packages with the same strings.

Reply:
Humm I think you did not saw I completely changed the PR. Check a look on the source code  tab. 

I won't do such kind of SQL requests because this not really the purpose of SQL, and this is harder to maintain. I kept the code, and added some logic to fix the issue. By the  way, as I commented, I found an issue in GLPI which required a workaround for this specific problem.

Reply:
as we talked let's merge the changes an leave the query for those tester that have a problem with the file paths